src/apps/front-office/home/pages/components/PopularProducts/PopularProductsTabs/PopularProductsTabs.tsx

```diff
@@ -0,0 +1,103 @@
+import { useEvent } from "@mongez/react-hooks";
+import { popularProductsAtom } from "apps/front-office/home/atoms/popular-products-atom";
+import { products } from "apps/front-office/utils/data";
+import { Product, ProductCategory } from "apps/front-office/utils/types";
+import { useState } from "react";
+
+export type PopularProductsTabsProps = {
+  children: React.ReactNode;
+};
+
+export default function PopularProductsTabs() {
+  const categoryTabs: ProductCategory[] = [
```

Comment:
I'd recommend moving the static array to be away from the component, and pass it to component directly, this will reduce the code update when API data is involved

Reply:
Done